packages/terra-functional-testing/src/services/wdio-visual-regression-service/VisualRegressionLauncher.js

```diff
@@ -1,83 +1,63 @@
-/* eslint no-unused-vars: ["error", { "args": "none" }] */
 /* global browser */
 import _ from 'lodash';
 import { parse as parsePlatform } from 'platform';
-import logger from '@wdio/logger';
 
+import { LocalCompare } from './compare';
 import makeElementScreenshot from './modules/makeElementScreenshot';
 import makeDocumentScreenshot from './modules/makeDocumentScreenshot';
 import makeViewportScreenshot from './modules/makeViewportScreenshot';
 
 import getUserAgent from './scripts/getUserAgent';
-import { mapViewports, mapOrientations } from './modules/mapViewports';
-
-const log = logger('wdio-visual-regression-service');
+import getTerraFormFactor from './modules/getTerraFormFactor';
 
 class VisualRegressionLauncher {
+  /**
+   * @param {Object} options - Service configuration options.
+   * @param {Object} options.baseScreenshotDir - The base screenshot directory path to save screenshot in
+   * @param {Object} options.locale - The locale being tested
+   * @param {Object} options.theme - The theme being tested
+   */
   constructor(options) {
-    this.options = options;
+    this.compare = new LocalCompare(options);
+    this.context = null;
     this.currentSuite = null;
     this.currentTest = null;
   }
 
-  /**
-   * Gets executed once before all workers get launched.
-   * @param {Object} config - wdio configuration object
-   * @param {[Object]} capabilities - list of capabilities details
-   */
-  async onPrepare(config) {
-    this.validateConfig(config);
-    this.compare = this.options.compare;
-    log.info('Launching onPrepare functions');
-    await this.runHook('onPrepare');
-  }
-
   /**
    * Gets executed before test execution begins. At this point you can access
    * all global variables, such as `browser`.
-   * It is the perfect place to define custom commands.
-   * @param  {object} capabilities - desiredCapabilities
-   * @param  {[type]} specs
-   * @return {Promise}
+   * @param {object} capabilities - desiredCapabilities
+   * @param {[type]} specs
+   * @return null
    */
-  async before(capabilities, specs) {
-    this.validateConfig(browser.config);
-
-    this.compare = this.options.compare;
-    this.viewportChangePause = _.get(browser.config, 'visualRegression.viewportChangePause', 100);
-    this.viewports = _.get(browser.config, 'visualRegression.viewports');
-    this.orientations = _.get(browser.config, 'visualRegression.orientations');
+  async before() {
     const userAgent = await browser.execute(getUserAgent);
     const { name, version, ua } = parsePlatform(userAgent);
 
     this.context = {
-      browser: {
+      browserInfo: {
```

Comment:
I changed this to browserInfo from browser because I was getting confused with the global browser object